adagucserverEC/CDataReader.cpp

```diff
@@ -297,6 +297,10 @@ bool CDataReader::copyCRSFromADAGUCProjectionVariable(CDataSource *dataSource, c
   // }
   dataSource->nativeProj4.copy(proj4Attr->toString().c_str());
 
+  // Fixes issue https://github.com/KNMI/adaguc-server/issues/279
+  dataSource->nativeProj4.replaceSelf("\"", "");
```

Comment:
I have asked on the proj mailing list and they are saying that this might change in the future. It is best only to remove the first " and ending " when the string starts with a ".